sos/report/plugins/ceph_common.py

```diff
@@ -17,7 +17,7 @@ class Ceph_Common(Plugin, RedHatPlugin, UbuntuPlugin):
     plugin_name = 'ceph_common'
     profiles = ('storage', 'virt', 'container')
 
-    containers = ('ceph-(mon|rgw|osd).*',)
+    containers = ('ceph-(.*)?(mon|rgw|osd).*',)
```

Comment:
Few other changes in other commits have `(.*-)` instead of just `(.*)`. Is that intentional?

Reply:
if ceph container is named differently (not beginning with ceph-) would there be an issue gathering the ceph stuff from such a container?

Reply:
Practically, with current naming schemes the `(.*)?` is sufficient, but `(.*-)?` is more exact. For consistency sake I'll sync all of these to be `(.*-)?`.

Reply:
> if ceph container is named differently (not beginning with ceph-) would there be an issue gathering the ceph stuff from such a container?

It means we won't activate the plugin based solely on the presence of the container, and would rely on the other enablement triggers. For the `ceph_common` plugin specifically there are no other triggers, so this would result in that plugin not running.